subsys/fw_info/Kconfig

```diff
@@ -31,9 +23,10 @@ config FW_INFO_FIRMWARE_VERSION
 	default 1
 
 config FW_INFO_MAGIC_COMMON
-	hex "Magic word for all structs (32 bits)."
+	hex
```

Comment:
These are now intended to be hidden right?

Reply:
Correct